packages/perseus/src/widgets/reaction-diagram.tsx

```diff
@@ -1,20 +1,17 @@
 import * as i18n from "@khanacademy/wonder-blocks-i18n";
-import PropTypes from "prop-types";
 import * as React from "react";
 
 import {Molecule} from "./molecule";
 
+import type {
+    PerseusReactionDiagramSeparators,
+    PerseusReactionDiagramWidgetOptions,
+} from "../perseus-types";
 import type {WidgetExports} from "../types";
 
-class Separator extends React.Component<any> {
+class Separator extends React.Component<PerseusReactionDiagramSeparators> {
```

Comment:
Some here: I'm not sure we really want to be using types from `perseus-types` as PropTypes, but I imagine I'm just wrong or overly cautious.